packages/contracts-bedrock/scripts/check-deploy-configs.sh

```diff
@@ -7,12 +7,14 @@
 code=$?
```

Comment:
Reviewers: I noticed this but didn't change it because I'm not sure of the use case of the script—perhaps this is supposed to refer to some other command that is supposed to be run beforehand? But I expect `$?` would always be zero here anyway